### ISSUE TITLE
Use "@" and "." in email address

### ISSUE THREAD

**Issue by brianyuen** [ptomaszek/dummy-form-filler#8]
Would love to see the "@" and ".com/net/org" set up for email fields to pass validation rules. Thank you for this great add-on!

**Comment by ptomaszek** [comment]
Hi,

Thanks for raising the issue.

String generated for the ```<input type="email">``` should pass email validation (the string will have the at-sign and a domain).
What's missing but will be introduced in the next version of the plugin is some additional semantic recognition, e.g. the plugin will be looking at a ```<label />``` and preceding free text that could describe the purpose of the input. For example, when it comes to emails, the following three inputs will be populated with some email addresses:
```
    <input id="email" type="email">
    <br>
    <label for="emailTextInput">'email' text inside the label</label> <input id="emailTextInput" type="text">
    <br>
    Just 'e-mail' string before the text input<input type="text">
```
This update will be released in a week or two with v0.12.

**Comment by ptomaszek** [comment]
Surprisingly, the new release has been approved very quickly so the update is already available.
